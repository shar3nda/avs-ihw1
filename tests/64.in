13
6 0 9 10 8 1 7 8 -2 10 5 8 3