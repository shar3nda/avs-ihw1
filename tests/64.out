-2 -1 9 6 7 -2 -1 6 9 -1